projects/documentation/content/guides/configuring-openwc.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: guide.njk
+title: 'Configuring your @open-wc project'
+displayName: Configuring your project
+slug: configuring-openwc
+---
+
+This guide expounds upon configuration specifics you may want to adopt in your `@open-wc` project for a smoother development experience that matches the specifications in Spectrum Web Components.
+
+## Building your project
+
+In your terminal, run the following command:
+
+```bash
+npm init @open-wc
+```
+
+Once you do that, `@open-wc` will give you a few configuration options. You'll want to set up your project as a Web Component and allow it to use typescript. When it asks you what features to add, say "yes" to all of them--we'll explain how to set them up to be more compatible with SWC in this guide. SWC uses `yarn` to manage its package dependencies, so you can go ahead and choose that one, as well.
```

Comment:
```suggestion
Once you do that, `@open-wc` will give you a few configuration options. You'll want to set up your project as a Web Component and allow it to use typescript. When it asks you what features to add, say "yes" (with the Space Bar, not the Enter key) to all of them--we'll explain how to set them up to be more compatible with SWC in this guide. SWC uses `yarn` to manage its package dependencies, so you can go ahead and choose that one, as well.
```
This is a common mistake with that generator, not sure if it's worth bringing up here, but it could be helpful.